connect-examples/v2/java_payment/README.md

```diff
@@ -25,7 +25,7 @@ Note that Apple Pay cannot be tested when running locally. You'll need to deploy
 ## Running locally
 
 The app can be run on a command line using Maven. The application expects three environment variables
-to be set: `SQUARE_APPLICATION_ID`, `SQUARE_LOCATION_ID` and `SQUARE_ACCESS_TOKEN`. Both of these can be copied from the
+to be set: `ENVIRONMENT`, `SQUARE_APPLICATION_ID`, `SQUARE_LOCATION_ID` and `SQUARE_ACCESS_TOKEN`. Both of these can be copied from the
 [Developer Dashboard](https://connect.squareup.com/apps). Keep in mind that the access token is
```

Comment:
nit: While they go to the exact same place, I think we're generally advertising `https://developer.squareup.com/apps` now